src/middlewared/middlewared/alert/source/active_directory.py

```diff
@@ -1,13 +1,38 @@
-from middlewared.alert.base import AlertClass, AlertCategory, AlertLevel, FilePresenceAlertSource
+from datetime import timedelta
+import os
+import logging
+from middlewared.alert.base import Alert, AlertLevel, ThreadedAlertSource
+from middlewared.alert.schedule import IntervalSchedule
 
+log = logging.getLogger("activedirectory_check_alertmod")
 
-class ActiveDirectoryDomainBindAlertClass(AlertClass):
```

Comment:
@themylogin can you verify this file please? It seems its reverting your alert class required changes?